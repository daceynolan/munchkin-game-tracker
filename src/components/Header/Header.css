.header {
  margin-top: 70px;
  min-width: 300px;
}
