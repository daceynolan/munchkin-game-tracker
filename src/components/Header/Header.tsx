import React from "react";

import logo from "../../images/logo-transparent.png";
import "./header.css";

const Header = () => {
  return (
    <div className="header">
      <img className="header__logo" src={logo} alt="Munchkin game logo" />
    </div>
  );
};

export default Header;
