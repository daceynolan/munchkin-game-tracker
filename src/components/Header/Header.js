import React from "react";

import logo from "../../images/logo-transparent.png";
import "./Header.css";

const Header = () => {
  return (
    <div className="header">
      <img className="header__logo" src={logo} alt="Munchkin game logo" />
    </div>
  );
};

export default Header;
