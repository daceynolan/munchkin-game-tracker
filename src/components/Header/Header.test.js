import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import Header from "./";

describe("<Header/>", () => {
  test("renders without crashing", () => {
    const { container } = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
