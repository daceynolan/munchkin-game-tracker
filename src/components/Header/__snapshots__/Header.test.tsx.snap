// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders without crashing 1`] = `
<div
  class="header"
>
  <img
    alt="Munchkin game logo"
    class="header__logo"
    src="logo-transparent.png"
  />
</div>
`;
